PDSim/scroll/core.py

```diff
@@ -15,7 +15,7 @@
 import warnings
 
 from CoolProp import State
-from math import pi
+from math import pi, exp
```

Comment:
This exp import isn't needed...

Reply:
actually it is. It is used inside heat_transfer_coefficient from Tagri and Jayaraman

Reply:
Ah, makes sense, but that isn't visible in the PR, so it seemed not used to me.